README.md

```diff
@@ -121,7 +121,7 @@ vars:
 ```  
 
 #### Change the build schema
-By default, this package builds the SAP staging models within a schema titled (`<target_schema>` + `_sap_source`) in your destination. If this is not where you would like your sap staging data to be written to, add the following configuration to your root `dbt_project.yml` file:
+By default, this package builds the SAP staging models within a schema titled (`<target_schema>` + `stg_sap`) in your destination. If this is not where you would like your sap staging data to be written to, add the following configuration to your root `dbt_project.yml` file:
```

Comment:
We callout the materialization schema for the staging models, but not the transform models. Can you update this to highlight both staging and transform and also update the below yml accordingly as well.

Reply:
@fivetran-joemarkiewicz  oh right! Updated this section.